java/src/gov/usgs/processingformats/Site.java

```diff
@@ -40,15 +42,32 @@
 	 */
 	private String location;
 
+	/**
+	 * Required Double containing the latitude
```

Comment:
expected units in comments is very helpful! I see it in Site.md, but seems meaningful in the code variable declaration too, what do you think?